deploy/manifests/kip/overlays/conformance-tests/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 bases:
 - ../../base
 configMapGenerator:
-- name: virtual-kubelet-config
+- name: kip-config
   namespace: kube-system
   behavior: merge
   files:
```

Comment:
This shows as a rename from deploy/manifests/virtual-kubelet/overlays/gcp/kustomization.yaml → deploy/manifests/kip/overlays/conformance-tests/kustomization.yaml for me for some reason.

Reply:
Never mind, I think it's due to how git diff shows the change.